documentation/messages/user.registration.transactional.md

```diff
@@ -94,10 +94,7 @@ Accessed by connecting to RabbitMQ server and sending message in the following f
   /* Required: user_id or email or mobile. */
   mobile: Integer,
 
-  /* Optional. A string to tag the message with that can be referenced in Mandrill. Example: `drupal_user_register`. */
-  registration_source: String,
-
-  /* Optional. Used to determine the origin of the request as a site in multisite environment. Default: `US`. */
+  /* Optional. Client application identifier. Example: `Phoenix-US`. */
```

Comment:
Oh, I actually wanted to use just `Phoenix` as an example. Yes, we're going to need change Phoenix, but I think sometime later we could strip Quicksilver functionality from it and make it use the API.